linux/sound.cpp

```diff
@@ -64,16 +61,9 @@ void CSound::OpenJack ( const bool bNoAutoJackConnect, const char* jackClientNam
     // check sample rate, if not correct, just fire error
     if ( jack_get_sample_rate ( pJackClient ) != SYSTEM_SAMPLE_RATE_HZ )
     {
-        throw CGenErr ( tr ( "The Jack server sample rate is different from "
-                             "the required one. The required sample rate is:" ) +
-                        " <b>" + QString().setNum ( SYSTEM_SAMPLE_RATE_HZ ) + " Hz</b>. " +
-                        tr ( "You can "
-                             "use a tool like <i><a href=\"https://qjackctl.sourceforge.io\">QJackCtl</a></i> "
-                             "to adjust the Jack server sample rate." ) +
-                        "<br>" +
-                        tr ( "Make sure to set the "
-                             "Frames/Period to a low value like " ) +
-                        QString().setNum ( DOUBLE_SYSTEM_FRAME_SIZE_SAMPLES ) + tr ( " to achieve a low delay." ) );
+        throw CGenErr ( QString ( tr ( "The JACK server isn't running at a sample rate of <b>%1 Hz</b>. Please use "
+                        "a tool like <i><a href=\"https://qjackctl.sourceforge.io\">QJackCtl</a></i> to set the "
+                        "the JACK server sample rate to %1 Hz." ) ) .arg ( SYSTEM_SAMPLE_RATE_HZ ) );
```

Comment:
If we have a general "Connecting Jamulus to JACK" page on the wiki, it might be worth linking that from each message, too.

Reply:
Yes. This would probably be something for the community KB?

Reply:
@jujudusud since you’re working on JACK and the documentation, I think that’s something for you?